lib/health_checks/custom/memory_check.rb

```diff
@@ -0,0 +1,17 @@
+module HealthChecks
+  module Custom
```

Comment:
All health checks are "custom", the naming of the module doesn't make sense. A better one would be Checks

Reply:
it was initially but HealthChecks::Checks::MongoidCheck seemed to me too checkish, so I changed it :) I'll put it back to Checks

Reply:
It is too check-ish but health checks is the name of the gem so the first "check" is OK, you can do MongoidCheck -> Mongoid

Reply:
I will leave it as it is since having MongoidCheck -> Mongoid will add some confusion with the mongoid gem. I tried it already with redis and it didn't look right